Example/Podfile

```diff
@@ -19,7 +19,7 @@ target 'Core_Example_iOS' do
   # The next line is the forcing function for the Firebase pod. The Firebase
   # version's subspecs should depend on the component versions in their
   # corresponding podspec's.
-  pod 'Firebase/CoreOnly', '5.20.1'
+  pod 'Firebase/CoreOnly', '5.20.990'
```

Comment:
Can you please add a comment here to change it back afterwards?

Reply:
The github branch/tag script will change these back automatically.

Reply:
Got it, thanks.